src/components/Card.tsx

```diff
@@ -50,29 +51,30 @@ type CardProps = StateProps & DispatchProps;
 export const Card = (props: CardProps) => {
     return (
         <View
-            style={styles.container}
             testID={'YourFeed/Post' + props.post._id}
+            style={{
+                paddingBottom: 12,
+            }}
         >
+            <View style={styles.container}>
+                <ActionsOverlay
+                    post={props.post}
+                    author={props.author}
+                    isSelected={props.isSelected}
+                    onDeletePost={props.onDeletePost}
+                    onSharePost={props.onSharePost}
+                    togglePostSelection={props.togglePostSelection}
+                />
 
-            <CardTop
-                post={props.post}
-                currentTimestamp={props.currentTimestamp}
-                author={props.author}
-                modelHelper={props.modelHelper}
-                navigation={props.navigation}
-                onSharePost={props.onSharePost}
-            />
-            <TouchableOpacity
-                activeOpacity={1}
-                onLongPress={() => props.togglePostSelection(props.post)}
-                onPress={() => openPost(props.post)}
```

Comment:
Opening the post by tapping it was removed. I think this is a regression unless we add another option to open it (e.g. a new action button).